zwave_js_server/event.py

```diff
@@ -1,21 +1,42 @@
 """Event base classes for Z-Wave JS."""
+from dataclasses import dataclass, field
+from typing import Any, Callable, Dict, List
+
+
+@dataclass
+class Event:
+    """Represent a Driver event."""
+
+    type: str
+    data: dict = field(default_factory=dict)
 
 
 class EventBase:
-    def __init__(self):
+    """Represent a base for event handling model classes."""
+
+    def __init__(self, protocol: Any) -> None:
```

Comment:
I'm not sure how to type the protocol in the best way since we want to be able to accept any model protocol Enum.